metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -650,6 +650,9 @@ final case class ScalaTestSuiteSelection(
 case class DebugUnresolvedTestClassParams(
     testClass: String,
     @Nullable buildTarget: String = null,
+    @Nullable jvmOptions: java.util.List[String] = null,
+    @Nullable env: java.util.Map[String, String] = null,
+    @Nullable envFile: String = null,
```

Comment:
Just a question.

Is it bloop only?
I haven't found these params in build-server-protocol

Reply:
no, it's both bloop and sbt (all build servers which support debugging) - https://github.com/build-server-protocol/build-server-protocol/pull/304

Reply:
thx for explanation!